docs/guides/preinstall.md

```diff
@@ -4,77 +4,111 @@ These steps are common to most distros. Please refer to both this page and this
 
-!!! hint "`dd`"
-    If you are familiar with the `dd` tool and the command line, you can use that instead. Use `diskutil list` to find the correct disk, then `sudo dd if=path/to/filename.iso of=/dev/rdiskX bs=1m`. Press `control-T` while it is running to make it show its current progress. **Make sure you select the correct disks.**
+Or you can use the macOS terminal:
 
-While the installation image is being written to the USB, you can [copy Wi-Fi firmware](#copy-wi-fi-firmware), but don't follow the steps after that until it has finished writing the image.
+1. Insert your USB drive.
+2. Open macOS Terminal.
+3. Run `diskutil list` to list all the drives.
+4. Look up for your USB thumb which appears as an **external, physical** drive labelled `/dev/diskX` where `X` is a single number (e.g. /dev/disk**2**).
+5. Run `sudo diskutil unmountDisk /dev/diskX` to unmount the disk.
+6. Run `sudo dd if=path/to/linux.iso of=/dev/rdiskX bs=1m` to start writing. To fetch the iso path easily you can just drag and drop the .iso into the Terminal.
+7. Now you may press `control-T` to make it show how many KBs it has written so far.
+
+While the installation image is being written to the USB, you can skip to [Copy Wi-Fi firmware](#copy-wi-fi-firmware); but don't follow the steps after it, wait until the ISO has been written to disk.
 
 ## Copy Wi-Fi firmware
 
-Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, so we copy these files from macOS to the EFI partition where Linux can access these files and then install them.
+Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, so we copy these files from macOS to the EFI partition where Linux can access and eventually install them.
+
+[Follow here](https://wiki.t2linux.org/guides/wifi-bluetooth/#on-macos) the instructions for the **first part done in macOS** and come back to this page.
 
-Follow the instructions in this section to be followed on macOS [here](https://wiki.t2linux.org/guides/wifi-bluetooth/#on-macos). The sections of that page to be followed on Linux can be followed later, once you've successfully installed Linux.
+The **second part "On Linux"** must be followed **after** you have completed the Linux installation.
 
 ## Disable Secure Boot
 
-!!! note
-    Wait until you have finished creating your install USB and you have copied Wi-Fi firmware before proceeding to this section.
+Now that you have completed the first part of the Wi-Fi firmware in macOS, you can proceed with the disabling of the secure boot.
+Apple's Secure Boot implementation does not allow booting anything other than macOS or Windows when it is enabled (not even shim signed GRUB).
+We need to disable it:
 
-Apple's Secure Boot implementation does not allow booting anything other than macOS and Windows when Secure Boot is enabled (not even shim signed GRUB).
+1. Turn off your Mac
+2. Turn it on and rapidly press and hold `Command-R` until the black screen flashes
```

Comment:
I don’t think “rapidly” word is required

Reply:
Removed [c7f57f2](https://github.com/t2linux/wiki/pull/447/commits/c7f57f2678d0ccc4500ffd1a4c80294b40962655)